lib/events/auditlog.go

```diff
@@ -802,6 +811,26 @@ func (l *AuditLog) periodicSpaceMonitor() {
 			// If used percentage goes above the alerting level, write to logs as well.
 			if usedPercent > float64(DiskAlertThreshold) {
 				l.log.WarnContext(l.ctx, "Free disk space for audit log is running low", "percentage_used", usedPercent)
+				if l.AlertHandler != nil {
```

Comment:
The file uploader should run for all Teleport services (including auth), so I think we can safely remove this alert. Otherwise we'll end up with duplicates